flamingpy/_version.py

```diff
@@ -15,4 +15,4 @@
 """Version number (major.minor.patch[label])"""
 
 
-__version__ = "0.8.2a5-dev2"
+__version__ = "0.8.2a5.dev3"
```

Comment:
@ilan-tz is there a reason why this is a dot `.` and the other is a hyphen `-`?

Reply:
@nariman87 mentioned the dot is more pythonic (and indeed matches [PEP 440](https://peps.python.org/pep-0440/). We'll use it from now on!

Reply:
Ah I see, in that case you can ignore my suggestion and merge it I think